### ISSUE TITLE
Support tvOS layered app icon images

### ISSUE THREAD

**Issue by adamkoch** [react-native-tvos/config-tv#20]
Is there a way to support tvOS layered app icon images right now? The config allows just a single .png when layered icons need 2-5 images.

https://developer.apple.com/design/human-interface-guidelines/app-icons#tvOS
https://developer.apple.com/design/human-interface-guidelines/images#Layered-images

Thanks!

**Comment by douglowder** [comment]
This can be done -- would have to give some thought to how to design the plugin properties to support both a single image and multiple layered images.

**Comment by filip-solecki** [comment]
Hey @douglowder !
Are there any updates? Is this library actively supported at all?

**Comment by douglowder** [comment]
Yes it is supported, and I will fix bugs. New features (like layered images) are taking a back seat to my other responsibilities...

**Comment by filip-solecki** [comment]
Sure, thanks

**Comment by filip-solecki** [comment]
Hey @douglowder - I worked on this and prepared a PR: [#40](https://github.com/react-native-tvos/config-tv/pull/40).
Can you take a look when you have a moment? 🙏 
Basically there are new options, `iconSmallLayers`, `iconSmall2xLayers` and `iconLayers`, next to the current single-image keys, each with a front, an optional middle and a back, taking precedence over the flat key for that icon. Configs that only use the flat keys stay unchanged.
Right now the plugin uses the same image for all three layers, which makes the stacking purely nominal in the compiled `Assets.car`. I've also prepared a minimal repo reproducing it: https://github.com/filip-solecki/config-tv-icon-layers-repro
